proto/viam/app/v1/app.proto

```diff
@@ -11,8 +11,38 @@ option go_package = "go.viam.com/api/app/v1";
 service AppService {
   rpc CreateLocation(CreateLocationRequest) returns (CreateLocationResponse);
 
+  // Organizations
+
+  // Create a new organization
+  rpc CreateOrganization(CreateOrganizationRequest) returns (CreateOrganizationResponse);
+
+  // List organizations
   rpc ListOrganizations(ListOrganizationsRequest) returns (ListOrganizationsResponse);
 
+  // Get an organization
+  rpc GetOrganization(GetOrganizationRequest) returns (GetOrganizationResponse);
+
+  // Update an organization
+  rpc UpdateOrganization(UpdateOrganizationRequest) returns (UpdateOrganizationResponse);
+
+  // Delete an organization
+  rpc DeleteOrganization(DeleteOrganizationRequest) returns (DeleteOrganizationResponse);
+
+  // List all members of an organization
```

Comment:
// List all members of an organization -> // List all members of an organization and all invited members to the organization.